test/integration/upload_access_test.rb

```diff
@@ -47,27 +43,22 @@ def assert_sent_private_upload(_upload, content_type)
     host! asset_host
   end
 
-  test 'allows everyone access to general uploads' do
+  test 'allows everyone to be redirected to general uploads' do
     upload = '/government/uploads/general-upload.jpg'
     create_uploaded_file(path_to_clean_upload(upload))
 
     get_via_nginx upload
 
-    assert_sent_public_upload upload, Mime[:jpg]
+    assert_redirected_to "/government/uploads/general-upload.jpg"
```

Comment:
I think this assertion might be slightly clearer if the `assert_redirected_to` included the asset host. As it stands, the test implies that you're always redirected to the same path that was requested.